linkerd/http/override-authority/src/lib.rs

```diff
@@ -23,6 +26,27 @@ pub struct OverrideAuthority<S, H> {
     inner: S,
 }
 
+/// Sets the [`Authority`][uri::Authority] of the given URI.
```

Comment:
optional: This can just be ```[`Authority`]``` since it's already imported, but I'm not sure what our style guidelines are around this.

Reply:
good catch, thanks! i appreciate the close eye. :eagle: 

this is fixed in the amended commit 5f9c146